plugin-unit-tests.md

```diff
@@ -24,17 +24,23 @@ We're going to assume that:
 
 So, let's get started:
 
-1. [Install PHPUnit](https://github.com/sebastianbergmann/phpunit#installation) (5.x is only supported when running php7, phpunit 4.8 is required when running php5).
+### 1. [Install PHPUnit](https://github.com/sebastianbergmann/phpunit#installation)
 
-1. Generate the plugin test files: 
+* [The latest version of WordPress supports PHPUnit 7.x](https://make.wordpress.org/core/handbook/testing/automated-testing/phpunit/#setup)
+* The legacy version of WordPress supported PHPUnit 5.x only when running PHP 7
+* The legacy version of WordPress required PHPUnit 4.8 when running PHP 5
```

Comment:
It would actually be helpful if the precise WordPress versions were mentioned, like "WordPress 5.5+ (is this correct) supports PHPUnit 7.x".

Reply:
I found Supported Version Chart:
[PHPUnit Compatibility and WordPress Versions – Make WordPress Core](https://make.wordpress.org/core/handbook/references/phpunit-compatibility-and-wordpress-versions/)

Is it better to replace all with link to this page?

Reply:
Yes, that way we don;t have to maintain the exact requirements in here.

Reply:
Updated.